code/modules/mob/living/living_defines.dm

```diff
@@ -162,7 +162,7 @@
 
 	///used for database logging
 	var/last_words
-
+	var/butt_sprite
```

Comment:
Autodoc; but also I don't think this var is super necessary. You could just override `get_butt_sprite()` when relevant, ie, 
```dm
/mob/living/silicon/get_butt_sprite()
    return BUTT_SPRITE_QR_CODE
```
This means admins can't mess with it but I don't know why admins would ever want to